experiments/base_experiment.py

```diff
@@ -649,7 +661,7 @@ def _save_model(self, filename=None):
         torch.save(
             {
                 "model": self.model.state_dict(),
-                "optimizer": self.optimizer.state_dict(),
+                "optimizer": self.optimizer.optimizer.state_dict() if self.cfg.training.optimizer=="LookAhead" else self.optimizer.state_dict(),
```

Comment:
I suspect that both the lookahead optimizer and the 'inner' optimizer have their own state dict, and we would have to save and load both in case we use the lookahead optimizer.